includes/UserMailer.php

```diff
@@ -849,16 +863,16 @@ function sendMails() {
 	 * @return Boolean
 	 * @private
 	 */
-	function sendPersonalised( $watchingUser ) {
+	private function sendPersonalised( $watchingUser ) {
 		global $wgContLang, $wgEnotifUseRealName;
 		// From the PHP manual:
 		//     Note:  The to parameter cannot be an address in the form of "Something <someone@example.com>".
 		//     The mail command will not parse this properly while talking with the MTA.
 		$to = new MailAddress( $watchingUser );
 
-		# $PAGEEDITDATE is the time and date of the page change
-		# expressed in terms of individual local time of the notification
-		# recipient, i.e. watching user
+		// $PAGEEDITDATE is the time and date of the page change
+		// expressed in terms of individual local time of the notification
+		// recipient, i.e. watching user
 		$body = str_replace(
```

Comment:
This logic is pretty much repeated twice, in a single function. It'd be nice to factor it out as well.